DevelopmentProposals/template.md

```diff
@@ -0,0 +1,61 @@
+## Project: TITLE
+***OpenSim Design Document***
+
+### Overview
+
+Just a few sentences that describes the purpose of this project. 
```

Comment:
Add to the overview section: "Describe the motivation for the change (e.g., reference an issue or describe benefits to the user)."